### PR TITLE
[roadmap] redesign

### PR DESCRIPTION
**Pull request by gyurka00**

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit d34fa0d7d9878a1f2d6ce3d3e9bdcef2c74f767a

https://app.netlify.com/sites/augmint/deploys/5a901cdafd0efa5a1654e293

**Comment by gyurka00** [comment]
@szerintedmi @reyrodrigez can you check it?

File: src/containers/roadmap/index.js [hunk 1]
```diff
@@ -4,30 +4,40 @@ import { states } from "./helpers.js";
...
+    StyleStateListItem
 } from './style';
 
-import roadmapLine from "assets/images/roadmap-line.svg";
-
 const statestLength = states.length;
 
 export default () => (
     <StyleRoadmap id="roadmap">
+        <StyleRoadmapTitle>ROADMAP</StyleRoadmapTitle>
```
Comment (reyrodrigez) [inline src/containers/roadmap/index.js]: Text should be converted to uppercase with css `text-transform: uppercase`

Comment (gyurka00) [reply src/containers/roadmap/index.js]: 👍 
This is a bad habit, I used it when we used semantic ui... 😅

**Pull request merged**